src/test/java/com/suse/saltstack/netapi/client/SaltStackClientTest.java

```diff
@@ -213,4 +213,24 @@ public void testStartCommand() throws SaltStackException {
         assertEquals(job.getJid(), "20150211105524392307");
         assertEquals(job.getMinions(), Arrays.asList("myminion"));
     }
+    
+    @Test
+    public void testStartCommandAsync() throws SaltStackException {
+    	 stubFor(post(urlEqualTo("/minions")).willReturn(
+                 aResponse().withStatus(HttpURLConnection.HTTP_OK)
+                         .withHeader("Content-Type", "application/json")
+                         .withBody(JSON_START_COMMAND_RESPONSE)));
+
+        Future<Job> future = client.startCommandAsync("*", "test.ping", null, null);
+        Job job;
+        try {
+        	job = future.get();
+        } catch (InterruptedException | ExecutionException e) {
+            throw new SaltStackException(e);
```

Comment:
I've tried to follow the unwritten conventions of the SaltStackClientTest class, the other test*Async methods do the same catch-wrap-throw thing, possibly to reduce the number of thrown exception of the test method. (As it is a unit-test method, I don't think it makes sense.)
it's a styling question, I can't judge it.
of course it can be removed and add these 2 exception types to the methods' "throws" list.

please someone tell me which one is preferred:
A, less, simplier code line (without the catch-wrap-throw) (should I change the other test*Async methods too? or should it be a separate PR?)
B, follow the style of the other *Async methods.

I would choose A, with or without modifying the other test*Async methods.